tests/sqlite/create.du

```diff
@@ -0,0 +1,44 @@
+from UnitTest import UnitTest;
+
+import "../../kahless.du" as kahless;
+
+@Table("users")
+class User < kahless.Model {
+    @Column("first_name")
+    var firstName = "";
+
+    @Column("last_name")
+    var lastName = "";
+
+    @Column("age")
```

Comment:
I've updated the code to note require all fields to have the Column annotation. Just throwing htat out there. :D